application/src/main/java/org/thingsboard/server/service/queue/processing/AbstractTbRuleEngineSubmitStrategy.java

```diff
@@ -51,7 +52,18 @@ public void update(ConcurrentMap<UUID, TbProtoQueueMsg<TransportProtos.ToRuleEng
         List<IdMsgPair<TransportProtos.ToRuleEngineMsg>> newOrderedMsgList = new ArrayList<>(reprocessMap.size());
         for (IdMsgPair<TransportProtos.ToRuleEngineMsg> pair : orderedMsgList) {
             if (reprocessMap.containsKey(pair.uuid)) {
-                newOrderedMsgList.add(pair);
+                var oldValue = pair.getMsg().getValue();
```

Comment:
oldValue needs to sound better to me. All around are values. I would like to have a descriptive variable name.

like `oldRuleEngineMsg` or any similar